libbeat/publisher/pipeline/output.go

```diff
@@ -45,7 +45,7 @@ type netClientWorker struct {
 
 func makeClientWorker(observer outputObserver, qu workQueue, client outputs.Client) outputWorker {
 	if nc, ok := client.(outputs.NetworkClient); ok {
-		c := &netClientWorker{observer: observer, qu: qu, client: nc}
+		c := &netClientWorker{observer: observer, qu: qu, client: nc, logger: logp.NewLogger("publisher_pipeline_output")}
```

Comment:
Nit: Now that there's 3 fields being initialized, consider breaking this up into multiple lines for readability.

Reply:
good point! will do! Thanks!